content/reference/admin/custom_sso/index.md

```diff
@@ -0,0 +1,94 @@
+We provide some pointers below about the supported SSO systems, but keep in mind that you will need to open a support
+ticket first before you start configuring anything on your end.
+{{< /alert >}}
+
+### OpenID Connect (OIDC) v1.0
+
+If your SSO system offers OIDC v1.0 integration, it will typically publish its metadata at a well known URL, often of
+the form `https://my-sso-server.com/.well-known/openid-configuration`. To configure the SSO integration, we will need:
+
+- the metadata URL
+- the client authentications which are supported by your OIDC implementation (client secret sent as POST, Basic Auth, or
+  JWT)
+- the client ID and secret we should use to connect to the SSO
+
+#### Azure Active Directory (Azure AD) using OIDC
+
+For instance, in Azure AD, create an app registration for Gatling Enterprise, and configure which accounts are allowed
+to connect. You will then find the metadata URL in Overview > Endpoints > OpenID Connect metadata document, and you can
+generate a new client secret in Certificates & secrets > Client secrets.
+
+We can then give you our redirect URI, which you will need to configure in Overview > Add a redirect URI.
+
+### SAML v2.0
+
+{{< alert tip >}}
+If your SSO system supports both OIDC and SAML, we recommend using OIDC.
+{{< /alert >}}
+
+If your SSO system offers SAML v2.0 integration, it will typically publish its Identity Provider (IdP) entity descriptor
+on a specific URL. To configure the SSO integration, we will need this IdP entity descriptor URL.
+
+We can then give our Service Provider (SP) descriptor if you need to import it into your SSO system.
+
+### Google
+
+See [Google's guide](https://support.google.com/cloud/answer/6158849) on setting up an oAuth 2.0 app. Make sure to:
+
+- select the user type "internal" to only allow users from your organization
+- add `gatling.io` as an authorized domain
+
+You will need to create a client ID of the type "web application", with the authorized redirect URI we will provide you,
+then send us the client ID and client secret.
+
+### GitHub
+
+See [GitHub's guide](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app) on setting up
+an oAuth 2.0 app for an organization. Make sure to:
+
+- configure `https://cloud.gatling.io` as the Homepage URL
+- configure the Authorization callback URL we will provide you
+
+You will need to generate a new client secret, then send us the client ID and client secret. We will also need your
+GitHub organization's name to restrict access to users from your organization.
+
+### GitLab
+
+See [GitLab's guide](https://docs.gitlab.com/ee/integration/oauth_provider.html) on setting up an oAuth 2.0 app for a
+group. Make sure to:
+
+- configure the redirect URI we will provide you
+- enable the following scopes: `api`, `read_user`, `read_api`
+
+You will need to send us the application ID and secret. We will also need your GitLab group ID to restrict access to
+users from your group.
```

Comment:
why a new line?

Reply:
To limit line lengths, but looking at the rest of this doc, it doesn't look like we did it elsewhere, with one or two exceptions. Will change then.

Reply:
Well, usually we split whole sentences. But it could be argued that since the parser squashes successive lines into a single proper paragraph, it could be a thing to force line breaks at a specific amount of characters!